src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -173,9 +174,8 @@ void readCastVoteRecords(List<CastVoteRecord> castVoteRecords, String contestId)
     }
   }
 
-  // parse the given file into a List of CastVoteRecords for tabulation
-  private void parseCvrFile(
-      String filePath, List<CastVoteRecord> castVoteRecords, String contestIdToLoad) {
+  // parse the CVR file or files into a List of CastVoteRecords for tabulation
+  private void parseCvrFile(List<CastVoteRecord> castVoteRecords, String contestIdToLoad) {
```

Comment:
I would swap the function names parseCvrFile / gatherCvrsForContest since 'gather is really handling per-file logic now.